app/views/map/map.html.erb

```diff
@@ -99,7 +99,7 @@
 
       L.LayerGroup.EnvironmentalLayers({
         // baseLayers: {
```

Comment:
@jywarren, @nstjean, We had initially passed this in because of the error in the map tests. These can be uncommented if needed after this PR is merged.  Thanks!